pkg/container-utils/containerutils.go

```diff
@@ -83,23 +83,23 @@ out:
 */
 import "C"
 
-var ContainerRuntimes = []string{
+var AvailableRuntimes = []string{
 	docker.Name,
 	containerd.Name,
 	crio.Name,
 }
 
-func NewContainerRuntimeClient(runtime string) (runtimeclient.ContainerRuntimeClient, error) {
+func NewContainerRuntimeClient(runtime, socketPath string) (runtimeclient.ContainerRuntimeClient, error) {
```

Comment:
I was using `endpoint` only for the CRIs to distinguish it from the Docker Engine API socket path but I think `socketPath` is just fine for all of them as you said. It will make things more readable.